lib/msal-common/src/response/ResponseHandler.ts

```diff
@@ -197,7 +197,17 @@ export class ResponseHandler {
             serverResponse.error_description ||
             serverResponse.suberror
         ) {
-            const errString = `${serverResponse.error_codes} - [${serverResponse.timestamp}]: ${serverResponse.error_description} - Correlation ID: ${serverResponse.correlation_id} - Trace ID: ${serverResponse.trace_id}`;
+            const errString = `Error(s): ${
+                serverResponse.error_codes || Constants.NOT_AVAILABLE
+            } - Timestamp: ${
+                serverResponse.timestamp || Constants.NOT_AVAILABLE
```

Comment:
if server doesn't provide timestamp, why not default to NOW (i.e. client timestamp)

Reply:
This is for server errors, not client errors. By leaving it as not-available, we know not to go looking for it in LogsMiner if a customer shares their error message with us.

Reply:
I think tools like LogsMiner adds a buffer to timestamps. Client timestamp is decent approx.